db/migrations/20200413164851_create_log_make.sql

```diff
@@ -0,0 +1,33 @@
+-- +goose Up
+CREATE TABLE maker.log_make
+(
+    id         serial primary key,
+    log_id     bigint  not null references public.event_logs (id) on delete cascade,
+    header_id  integer not null references public.headers (id) on delete cascade,
+    address_id integer not null references public.addresses (id) on delete cascade,
+    offer_id   numeric,
+    pair       character varying(66),
```

Comment:
`pair` on LogKill is the same as it is here on LogMake where it's `pay_gem` and `buy_gem` packed together. Storing it as is as a varchar makes sense to me, but was just curious if you were knew why pair was stored like that, since we have `pay_gem` and `buy_gem` already on this event. 🤔

Reply:
My only thought is maybe a user would want to filter logs by the pair and hashing provides a consistent data structure that's easy to reproduce in various clients/ecosystems but I'm not sure 🤔